fabric-events-interaction-v0/src/main/java/net/fabricmc/fabric/mixin/eventsinteraction/MixinClientPlayerInteractionManager.java

```diff
@@ -57,7 +57,6 @@ public void attackBlock(BlockPos pos, Direction direction, CallbackInfoReturnabl
 		ActionResult result = AttackBlockCallback.EVENT.invoker().interact(client.player, client.world, Hand.MAIN_HAND, pos, direction);
 		if (result != ActionResult.PASS) {
 			info.setReturnValue(result == ActionResult.SUCCESS);
-			info.cancel();
```

Comment:
Might be best to keep the PR simple, and not go into other misc changes,

Reply:
Done